linkis-public-enhancements/linkis-datasource/linkis-metadata-query/common/src/main/java/org/apache/linkis/metadata/query/common/domain/MetaColumnInfo.java

```diff
@@ -30,6 +30,24 @@ public class MetaColumnInfo implements Serializable {
     private boolean primaryKey;
     private String name;
     private String type;
+    private int length;
+    private String colComment;
+
+    public String getColComment() {
+        return colComment;
+    }
+
+    public void setColComment(String colComment) {
+        this.colComment = colComment;
+    }
+
+    public int getLength() {
+        return length;
+    }
+
+    public void setLength(int length) {
+        this.length = length;
+    }
```

Comment:
After this DTO class MetaColumnInfo is changed, the compatibility with the client needs to be considered for class `org.apache.linkis.datasource.client.response.MetadataGetColumnsResult`, whether it is better to provide a unit test to verify the compatibility, specify the linkis-datasource-client that depends on version 1.1.2.